buyButton/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+body {
+    background: #3c4146;
+    font-family: sans-serif;
+    font-size: 30px;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.buy-button {
+    color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.total-price {
+    background: #00f5c3;
+    color: #3e4246;
+    height: 60px;
+    box-sizing: border-box;
+    border-bottom: 3px solid #00d7aa;
+    border-radius: 6px;
+    padding: 0 20px;
+    box-shadow: 1px 0px 19px rgba(0,245,195,0.769);
+    margin-left: 5px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+#count-box {
+    overflow: hidden;
+    position: relative;
+    width: 60px;
+    height: 60px;
+    border-radius: 10px;
+    background: #32373c;
+    box-shadow: inset 1px 3px 2px rgba(0,0,0,0.4);
+}
+span.line {
+    width: 30px;
+    height: 3px;
+    background: #fff;
+    display: inline-block;
+}
+span.line.line-plus-2 {
+    position: absolute;
+    transform: rotate(90deg);
+}
+.minus, .plus {
```

Comment:
please write this as 

````
.minus, 
.plus {
````
for readablity

Reply:
Done,
I usually do it like that , i will try from now on to to it always like so